app/UserPreferences.m

```diff
@@ -19,7 +19,7 @@
 static NSString *const kPreferenceDisableDimmingKey = @"Disable Dimming";
 NSString *const kPreferenceLaunchCommandKey = @"Init Command";
 NSString *const kPreferenceBootCommandKey = @"Boot Command";
-
+NSString *const kPreferenceHideStatusBar = @"Status Bar";
```

Comment:
```suggestion
NSString *const kPreferenceHideStatusBar = @"Status Bar";

```

Reply:
What am I supposed to change here? Just remove it?

Reply:
Nope, just keep the newline after the constants